Source/WebCore/dom/Microtasks.h

```diff
@@ -68,6 +68,8 @@ class MicrotaskQueue final {
     bool isEmpty() const { return m_microtaskQueue.isEmpty(); }
     bool hasMicrotasksForFullyActiveDocument() const;
 
+    static void runJSMicrotask(JSC::JSGlobalObject*, JSC::VM&, JSC::QueuedTask&);
```

Comment:
It seems that this function can stay private now?

Reply:
Unfortunately, it is still used in JSExecState and it is only used for UserGestureIndicator case. We would like to remove that and move the code to MicrotaskQueue (so it becomes centralized place to dispatch microtasks) so that we can remove this :)